test/integration/aaa_download_only_test.go

```diff
@@ -131,17 +131,6 @@ func TestDownloadOnly(t *testing.T) {
 				}
 			})
 
-			t.Run("binaries", func(t *testing.T) {
```

Comment:
how about instead of just deleting this test, we check for the preloadExists boolean and skip if it's true?

Reply:
Done. Reverted the removal commit, and added the skip.